music21/expressions.py

```diff
@@ -1032,15 +1442,53 @@ class Turn(Ornament):
     * Changed in v7: size is a Generic second.  removed unused nachschlag component.
     * Changed in v9: Added support for delayed vs non-delayed Turn.
```

Comment:
update docs.

Reply:
Still need to do this in a few places, I think.

Reply:
Done.